routers/admin_route.js

```diff
@@ -3,8 +3,13 @@ const suspendController = require("../controllers/admin_controller")
 
 const adminRoute = new express.Router()
 
-adminRoute.post('/suspension', suspendController.handleSuspend)
-adminRoute.post('/addUser', suspendController.handleAddUser)
-adminRoute.get('/showUser', suspendController.handleShowUser)
-adminRoute.delete('/deleteUser/email:', suspendController.handleDeleteUser)
+adminRoute.post('/suspension', suspendController.handleSuspend);
+adminRoute.post('/addUser', suspendController.handleAddUser);
+
+adminRoute.get('/showAllUsers', suspendController.handleShowAllUsers);
+adminRoute.get('/showUser/:email', suspendController.handleShowUser);
```

Comment:
please go over the path names and RESTful API conventions (learned in class 2) 
API path should be simple- 
- `PUT: /admin`
- `DELETE: /user`
- `GET: /users`